frontend/src/components/IngredientsAdder.js

```diff
@@ -77,20 +77,21 @@ export default function IngredientsAdder({
             }}
           >
             <option>Hozzávaló neve</option>
-            {ingredientsList
-              .filter(
-                (ingredient) =>
-                  !exludedIngredients
-                    .map(
-                      (excludedIngredient) => excludedIngredient.ingredient.id
-                    )
-                    .includes(ingredient.id)
-              )
-              .map((l) => (
-                <option key={l.id} value={JSON.stringify(l)}>
-                  {translateIngredient(l.name, l.measurement)}
-                </option>
-              ))}
+            {ingredientsList.sort((a, b) => a.name.localeCompare(b.name)) &&
+              ingredientsList
+                .filter(
+                  (ingredient) =>
+                    !exludedIngredients
+                      .map(
```

Comment:
Rövidebb lenne egyébként some-t használva, így: 
!exludedIngredients.some(excluded  => excluded.id === ingredient.id)

Reply:
done in  #118 #117